src/main/java/org/ays/auth/service/impl/AysRoleUpdateServiceImpl.java

```diff
@@ -63,6 +64,26 @@ public void update(final String id,
         roleSavePort.save(role);
     }
 
+    /**
+     * Activates an existing role.
+     * <p>
+     * This method sets the status of the role identified by its ID to active. If the role does not exist,
+     * an exception is thrown.
+     * </p>
+     *
+     * @param id The ID of the role to activate.
+     * @throws AysRoleNotExistByIdException if a role with the given ID does not exist.
+     */
+    @Override
+    public void activate(String id) {
+        final AysRole role = roleReadPort.findById(id)
+                .orElseThrow(() -> new AysRoleNotExistByIdException(id));
```

Comment:
Rol'ün pasif olmadığı için güncellenemediğine dair bir hata fırlatmamız iyi olabilir, çünkü pasif olmayan rolleri aktifleştirememeliyiz

Reply:
Güncelledim.